docs/version0.15.md

```diff
@@ -74,6 +74,9 @@ This option was redundant and has now been removed. If you try to use this optio
 
 `JVMJ9VM007E Command-line option unrecognised: -Xdiagnosticscollector`
 
+### Heuristics for compaction during idle GC
+OpenJ9 now automatically compacts the heap when certain triggers are met during idle GC. As a result of this change, [`-XX:IdleTuningCompactOnIdle`](xxidletuningcompactonidle.md) has been deprecated.
```

Comment:
Should we be more specific and mention `-XX:[+|-]IdleTuningCompactOnIdle`.

Reply:
Fixed